frontend/packages/client/jsconfig.json

```diff
@@ -1,7 +1,10 @@
 {
   "compilerOptions": {
-    "baseUrl": "src"
+    "baseUrl": "src",
+    "jsx": "react-jsx"
```

Comment:
add `jsx: react-jsx` so we dont have to `import React from 'react'` everywhere just to use JSX